build-scripts/android/build.sh

```diff
@@ -2,11 +2,11 @@
 
 set -e
 cd android
-./gradlew check
+./gradlew app:check
 
 if [[ $BUILD_RELEASE == 1 ]]; then
     ./gradlew assembleRelease --info --console plain | tee
 else
-    ./gradlew build --info --console plain | tee
```

Comment:
I have been testing and the metrics are recorded successfully both in android and iOS \o/
I only have a few questions and would prefer to have the other metrics in the code till we compare both and agree on remove them. 
That would be all and this could land :)